help_to_heat/templates/frontdoor/park-home.html

```diff
@@ -1,23 +1,23 @@
-{% extends "frontdoor/base.html" %}
-{% import "macros.html" as macros with context %}
-
-{% block title %}
-<title>{{_("Do you live in a park home? - Check eligibility for Great British Insulation Scheme - GOV.UK")}}</title>
-{% endblock %}
-
-{% block content %}
-  {% call macros.form_wrapper(prev_url) -%}
-    <div class="govuk-form-group {% if errors %}govuk-form-group--error{% endif %}">
-      <fieldset class="govuk-fieldset" {% if errors %} aria-describedby="question-park_home-error" {% endif %}>
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-          <h1 class="govuk-fieldset__heading">
-            {{_("Do you live in a park home?")}}
-          </h1>
-        </legend>
-
-        {{macros.radios('park_home', park_home_options_map, data, errors)}}
-
-      </fieldset>
-    </div>
-  {%- endcall %}
+{% extends "frontdoor/base.html" %}
```

Comment:
there is a line ending change here CRLF -> LF, though most other templates are LF so this seems fine